### DISCUSSION TITLE
Devtool Close Button

### DISCUSSION THREAD

**Discussion by IdoBouskila** [TanStack/query#6347] [Ideas] [answered]
Hello 👋🏼
I started to work with `@tanstack/query` new version (v5). Awesome work!
I notice the close button of the devtool is maybe too small which makes it harder to use and hurt the UX?

![image](https://github.com/TanStack/query/assets/94786579/2019d44c-2937-416a-9998-f13d6bc84519)

What do you think?

BTW, I'd be happy to send a PR for that :)

Thanks you!

**Comment by TkDodo** [comment, marked as answer]
you can also click the `TanStack` logo / text on the top left to close the devtools
